bank_accounts.rb

```diff
@@ -43,8 +48,8 @@ def self.get_ultimate_list
     end
 
     def withdraw(amount)
-      if (@balance < amount)
-        puts "This will result in a negative account balance. Transaction terminated."
+      if (@balance - amount < @min_balance)
+        puts "Your balance cannot be below the minimum amount of $#{@min_balance/100}. Transaction terminated."
```

Comment:
It seems like you use the `@min_balance/100` a few different times to get the "output" version of the balance so it may be worth having a separate method to do this for you so if you need to do this again later you'll be able to reuse the same functionality